examples/all-clusters-app/nrfconnect/configuration/mcuboot_multi_image_dfu.conf

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
You may synchronize with Arek and apply his changes to this config if he's done with his part.